### PR TITLE
chore(all): update CODEOWNERS

### PR DESCRIPTION
**Pull request by telpirion**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -8,19 +8,80 @@
...
+/getting-started/                     @GoogleCloudPlatform/go-samples-reviewers
+/internal/                            @GoogleCloudPlatform/go-samples-reviewers
+/testing/                             @GoogleCloudPlatform/go-samples-reviewers
 
+# CNDB, Storage, and Infra-DB
 /bigtable/                            @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/cloud-native-db-dpes
 /cloudsql/                            @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/infra-db-dpes
 /datastore/                           @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/cloud-native-db-dpes
 /firestore/                           @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/cloud-native-db-dpes
+/spanner/                             @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/infra-db-dpes
```
Comment (enocom) [inline .github/CODEOWNERS:21]: Could we make these infra-db-sdk while you're here? We renamed the team.

Comment (enocom) [reply .github/CODEOWNERS:21]: And technically, spanner should be under the "does not have an owner" category. We never touch Spanner stuff.

**Review by muncus — APPROVED** [review]
i've given `infra-db-sdk` access to this repo to match `infra-db-dpes`, since the group members are the same.

it looks like `mpwarres` does not have write access. @telpirion please either add permissions, or remove them from owners.

**Comment by telpirion** [comment]
I've sent mpwarres an invitation. In many cases, I tried to match ownership of folders to the owners defined for `python-docs-samples`. I figure that those folks would know to whom Go sample issues should be routed.

**Review by enocom — APPROVED** [review]

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -8,19 +8,80 @@
...
 /bigtable/                            @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/cloud-native-db-dpes
-/cloudsql/                            @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/infra-db-dpes
+/cloudsql/                            @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/infra-db-sdk
 /datastore/                           @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/cloud-native-db-dpes
 /firestore/                           @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/cloud-native-db-dpes
 /storage/                             @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/cloud-storage-dpes
 /storagetransfer/                     @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/cloud-storage-dpes
-/iot/                                 @GoogleCloudPlatform/go-samples-reviewers @gcseh @GoogleCloudPlatform/api-iot
 
+# AAP
```
Comment (iennae) [inline .github/CODEOWNERS:24]: can these all change over to torus-dpe please :)

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -8,19 +8,80 @@
...
 /appengine/                           @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/aap-dpes
 /appengine_flexible/                  @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/aap-dpes
+/container_registry/                  @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/aap-dpes 
 /eventarc/                            @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/aap-dpes
 /functions/                           @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/aap-dpes
 /run/                                 @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/aap-dpes
+/secretmanager/                       @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/aap-dpes
```
Comment (iennae) [inline .github/CODEOWNERS:31]: not sure where secret manager samples fit in terms of scope.

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -8,19 +8,80 @@
...
 /tasks/                               @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/aap-dpes
 /internal/cloudrunci/                 @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/aap-dpes
+
+# Data & AI DEE
+/automl/                               @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/dee-data-ai 
+/bigquery/                             @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/dee-data-ai
+/datacatalog/                          @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/dee-data-ai 
+/dataflow/                             @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/dee-data-ai @davidcavazos
+/dataproc/                             @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/dee-data-ai
+/dialogflow/                           @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/dee-data-ai @nicain
```
Comment (iennae) [inline .github/CODEOWNERS:41]: wondering why we have individuals in addition to teams defined in here

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -8,19 +8,80 @@
 # The go-samples-reviewers team is the default owner for anything not
 # explicitly taken by someone else.
 *                                     @GoogleCloudPlatform/go-samples-reviewers
+/docs/                                @GoogleCloudPlatform/go-samples-reviewers
+/getting-started/                     @GoogleCloudPlatform/go-samples-reviewers
```
Comment (iennae) [inline .github/CODEOWNERS:12]: it feels like getting-started is owned by all DEE as well.

File: .github/CODEOWNERS (hunk 2 above)
Comment (telpirion) [reply .github/CODEOWNERS:24]: Done.

File: .github/CODEOWNERS (hunk 3 above)
Comment (telpirion) [reply .github/CODEOWNERS:31]: I've moved secretmanager to the "does not have an owner" section

File: .github/CODEOWNERS (hunk 4 above)
Comment (telpirion) [reply .github/CODEOWNERS:41]: This mirrors the CODEOWNERS in the `python-docs-samples`, where some products have individuals defined as owners. I'm happy to remove the individuals if you prefer.

File: .github/CODEOWNERS (hunk 5 above)
Comment (telpirion) [reply .github/CODEOWNERS:12]: Good point. It doesn't look like there's a `dee-all` team in GoogleCloudPlatform. There is a `cloud-dpe` though--should I set that as CODEOWNER for /getting-started/ ?

File: .github/CODEOWNERS (hunk 4 above)
Comment (iennae) [reply .github/CODEOWNERS:40]: removing individuals would be preferred as it's extra to maintain as people move responsibilities. Thank you!

File: .github/CODEOWNERS [hunk 6]
```diff
@@ -8,19 +8,80 @@
...
+/securitycenter/                       @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/dee-infra 
+
+# DEE Platform Ops & Observability
+/container/                            @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/dee-platform-ops
+/logging/                              @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/dee-observability 
+/monitoring/                           @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/dee-observability
+/trace/                                @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/dee-observability @ymotongpoo 
+
+# Non-DEE owner
+/iot/                                  @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/api-iot @gcseh
```
Comment (iennae) [inline .github/CODEOWNERS:67]: we got to remove @gcseh from this line to fix this being broken currently.

**Review by iennae — APPROVED** [review]
one fix needed for sure for the individual defined, the others are nice to haves.

File: .github/CODEOWNERS (hunk 6 above)
Comment (telpirion) [reply .github/CODEOWNERS:67]: Done.

File: .github/CODEOWNERS (hunk 4 above)
Comment (telpirion) [reply .github/CODEOWNERS:40]: Good point. Done.

**Comment by telpirion** [comment]
CODEOWNERS should be a safe change 🤞🏻

**Pull request merged**